README.md

```diff
@@ -113,7 +116,7 @@ grunt custom --amd=""
 
 #### Custom Build Examples
 
-To create a custom build, first check out the version:
+To create a custom build, first check out the build version:
```

Comment:
this is not a version of the build but version of jQuery; we're checking out a tag in the repository.